Tests/AsyncHTTPClientTests/HTTPClientTestUtils.swift

```diff
@@ -19,6 +19,22 @@ import NIOConcurrencyHelpers
 import NIOHTTP1
 import NIOHTTPCompression
 import NIOSSL
+import NIOTransportServices
+
+/// Are we testing NIO Transport services
+func isTestingNIOTS() -> Bool {
+    return ProcessInfo.processInfo.environment["ENABLE_TS_TESTS"] == "true"
```

Comment:
@adam-fowler one last question: Could we flip the default here and have the tests run on TS by default? Otherwise I'm sure nobody will ever run them :|

Reply:
Done